### DISCUSSION TITLE
Edit recorded asciicasts (cut and concatenate parts)

### DISCUSSION THREAD

**Discussion by adamschmideg** [asciinema/discussions#55] [Ideas]
This is a feature request [I started to work on](https://github.com/adamschmideg/asciinema/tree/feature/editing). The idea is, the user edits the recorded asciicasts and changes the event types. Take for example this recording with edited event types,
```
[1, "o", "Hel"]
[2, "s", "l"]
[3, "s", "o"]
[4, "d", " whirled"]
[5, "o", " world"]
```
The extended event types are
- "s": squeeze with the previous event
- "d": delete this event

The result of the above example is
```
[1, "o", "Hello"]
[5, "o", " world"]
```
Editing would be invoked `asciinema edit --from part1.cast --from part2.cast result.cast`. Each input file is processed the way detailed above, then concatenated into a single cast file with the right offset.

**Comment by davedittrich** [comment]
This is similar to https://github.com/asciinema/discussions/issues/54.

I agree editing capabilities would be very helpful.
